src/main/java/org/jenkinsci/test/acceptance/po/FreeStyleMultiBranchJob.java

```diff
@@ -14,4 +15,28 @@
     public FreeStyleMultiBranchJob(Injector injector, URL url, String name) {
         super(injector, url, name);
     }
+
+    @Override
+    public <T extends BuildStep> T addBuildStep(Class<T> type) {
+        return addStep(type, "builder");
+    }
+
+    @Override
+    public <T extends PostBuildStep> T addPublisher(Class<T> type) {
+        T p = addStep(type, "publisher");
+
+        publishers.add(p);
+        return p;
+    }
+
+    private <T extends Step> T addStep(Class<T> type, String section) {
+        ensureConfigPage();
+
+        control(by.path("/projectFactory/hetero-list-add[%s]", section)).selectDropdownMenu(type);
+        waitFor(by.xpath("//div[@name='%s']", section), 3); // it takes some time until the element is visible
```

Comment:
@olivergondza No, it wasn't the only reason, but you are right in your both comments, I will create a follow up PR with your suggestions